### ISSUE TITLE
Glow is always on

### ISSUE THREAD

**Issue by soniktrooth** [craigh411/vue-star-rating#50]
it seems to me that the glow is always on. 

Given the follow set up
```
          <star-rating
            v-model="rating"
            v-if="rating !== null"
            :read-only="true"
            :increment="0.1"
            active-color="#d6612d"
           text-class="rating__text"
            :star-size="20"
          >
```
I get the following result:

![image](https://user-images.githubusercontent.com/597617/60555554-4f6be300-9cf2-11e9-8b2a-6131dd0052d9.png)

Explicitly setting `:glow="0"` doesn't change the outcome which is to be expected as the default is `0` anyway.

Setting the glow color explicitly shows that it is in fact the glow showing through despite 'glow' being set to o`0`

![image](https://user-images.githubusercontent.com/597617/60555433-a02f0c00-9cf1-11e9-8ec7-d49d35a010e9.png)

**Comment by decowboy** [comment]
I can confirm I am experiencing the same issue.

**Comment by 0x46** [comment]
While not ideal, I found that setting glow-color="transparent" works for me

**Comment by craigh411** [comment]
Fixed in version 1.6.3
